main/java/dao/UserDaoImpl

```diff
@@ -0,0 +1,30 @@
+package dao;
+
+import db.DataBaceConnetcion;
+import model.UserModel;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+
+public class UserDaoImpl implements UserDao {
+    private EntityManagerFactory connection;
+
+    public UserDaoImpl() {
+        connection = DataBaceConnetcion.getConnection();
+    }
+   @Override
+    public void addUser(UserModel user) {
+        EntityManager entityManager = connection.createEntityManager();
+        entityManager.getTransaction().begin();
+        entityManager.persist(user);
+        entityManager.getTransaction().commit();
+        entityManager.getTransaction().rollback();
```

Comment:
we need to use this method if we have exception when work with transaction, and most often it used in block catch. So better remove it from this method, because we don't use try-catch here)